### PR TITLE
changed multiple things, check commit description

### PR DESCRIPTION
**Pull request by VanshilShah**
added network manager, added location manager, changed main activity to have a simple map fragment, and wait until map is ready and location is ready before init.

### CODE REVIEW

**Comment by s7092910** [comment]
@VanshilShah You shouldn't use a fragment in the xml layout, it can cause stability issues. This is why many android developers tend to use a FrameLayout in the xml and then use that as the fragment to be replaced.

**Comment by VanshilShah** [comment]
@s7092910 That makes sense! I'll change it soon.

**Comment by omkarmoghe** [comment]
@VanshilShah @s7092910  Agreed, keep all the map stuff from the original repo and just PR for the network changes. Let's focus on one thing at a time for a PR

**Comment by VanshilShah** [comment]
it should be alright to merge now, I have changed the map stuff back to original, only change to the map stuff is that Locations are now handled by LocationManager

File: app/src/main/java/com/omkarmoghe/pokemap/network/NianticManager.java [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+        };
+        Call<NianticService.InitialResponse> call = nianticService.login();
+        call.enqueue(initialCallback);
+    }
+
+    private void retrofitLoginStep2(NianticService.LoginRequest loginRequest){
+        Callback<NianticService.LoginResponse> loginCallback = new Callback<NianticService.LoginResponse>() {
+            @Override
+            public void onResponse(Call<NianticService.LoginResponse> call, Response<NianticService.LoginResponse> response) {
+                //TODO: make the next call to finish getting the token.
```
Comment (omkarmoghe) [inline app/src/main/java/com/omkarmoghe/pokemap/network/NianticManager.java:111]: wait so has this been implemented?

wait nvm i see it below. this is where it's going to go haha

**Pull request merged**